code/bignum/Hacl.Bignum.MontArithmetic.fsti

```diff
@@ -21,6 +21,8 @@ module BM = Hacl.Bignum.Montgomery
 
 #set-options "--z3rlimit 50 --fuel 0 --ifuel 0"
 
+val _align_fsti : unit
```

Comment:
What is that for?

Reply:
There are some issues compiling this file in command-line mode otherwise. To be more precise, it seems that in command-line, the `‘freeable` definition introduced in the .fst [here](https://github.com/project-everest/hacl-star/blob/1ecddaafd4cde2d01677599a1f843619fc4933b1/code/bignum/Hacl.Bignum.MontArithmetic.fsti#L90) doesn't shadow the `Lib.Buffer.freeable` definition soon enough. Introducing this kind of "aligners" solves the issue.